docs/blog/2019-07-03-customizing-styles-in-gatsby-themes-with-theme-ui/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Customizing Styles in Gatsby Themes with Theme UI"
+date: 2019-07-03
+author: Brent Jackson
+tags:
+  - themes
+  - css
+  - styles
+  - theme-ui
+---
+
+With Gatsby themes we want to make the experience of building a site as flexible and extensible as possible.
+Themes allow you to reuse site configurations, sets of plugins, and components across multiple Gatsby sites.
+We've tried to design the API of themes to [_progressively disclose complexity_][progressive disclosure],
+where smart defaults are wrapped into abstractions, but customization and configuration are always available at your fingertips.
+The styles used in our official themes are no exception.
+
+## Say hello to our official themes
+
+Today, we're releasing two official themes: a blog theme inspired by Dan Abromov's [overreacted.io][], and our notes theme,
+which is intended for freeform, non-linear content and inspired by Tom Critchlow's concept of a _[Digital Garden][]_.
+Both themes include their own styles so that you can build and publish a blog without starting from scratch.
+In order to make these themes truly _themeable_, the values used for typography, color, layout, and other design tokens
+can be customized by editing part or all of a `theme` configuration object.
+This also means you can create and publish your own theme based on the official one,
+with custom typography and colors, by editing a few plain objects instead of recreating entire components.
```

Comment:
I don't have a CS background so I might be wrong here, and I definitely would like this to be as clear as possible, but I think of JSON as the language-agnostic syntax, whereas *plain object* was meant as a reference to Plain Old JavaScript Object (POJO) – i.e. not a function or other object type. We do want these to be JSON-serializable, but that's not a hard requirement. Glancing at both the Emotion and Styled Components docs, they simply refer to `theme` as an object. 

The wording does sound awkward here though, do you think something like this would be better?
```suggestion
with custom typography and colors, by editing an object instead of recreating entire components.
```

Reply:
I think `an object` sounds fine!